### DISCUSSION TITLE
页面卡顿无法加载

### DISCUSSION THREAD

**Discussion by GUSHUMING** [milvus-io/milvus-insight#235] [Q&A]
三千万数据集加载时，无法显示数据集情况，一开始页面能正常显示
![efc1c750b8440780fd46dc6265d1484](https://user-images.githubusercontent.com/73211918/134306095-0ea51d2c-e740-465f-9a67-999e403841ec.png)
点开详情就会
![38ea2506aaa9944b0a0d70fdea30e20](https://user-images.githubusercontent.com/73211918/134306183-be5fdbc4-f5cf-4597-9e45-8f069dbc7cf5.png)
显示空白，然后就有
![4353e065662d74ef83ceb62455b9304](https://user-images.githubusercontent.com/73211918/134306238-1bdf9d75-db67-4983-bd0d-e000f4af85eb.png)

**Comment by shanghaikid** [comment]
Sorry, We are not ready to support 3kw data amount for milvus insight.  
It will be resolved soon.
